src/matmul.jl

```diff
@@ -713,12 +776,26 @@ Base.@constprop :aggressive function syrk_wrapper!(C::StridedMatrix{T}, tA::Abst
         if (alpha isa Union{Bool,T} &&
                 beta isa Union{Bool,T} &&
                 stride(A, 1) == stride(C, 1) == 1 &&
-                _fullstride2(A) && _fullstride2(C))
+                _fullstride2(A) && _fullstride2(C)) &&
+                max(nA, mA) ≥ 4
             return copytri!(BLAS.syrk!('U', tA, alpha, A, beta, C), 'U')
+        else
+            return copytri!(generic_syrk!(C, A, false, tA_uc == 'N', alpha, beta), 'U')
         end
     end
```

Comment:
```suggestion
            BLAS.syrk!('U', tA, alpha, A, beta, C)
        else
            generic_syrk!(C, A, false, tA_uc == 'N', alpha, beta)
        end
        copytri!(C, 'U')
        return C
    end
```
I'd suggest this small change to make the branches a little simpler, and a similar change to `herk_wrapper!`

Reply:
sure, done